src/core/js/models/questionModel.js

```diff
@@ -40,55 +40,30 @@ define([
         setupButtonSettings: function() {
             var globalButtons = Adapt.course.get("_buttons");
```

Comment:
just a recommendation and not a requirement... but would be a good opportunity to use `const` here instead of `var`